pkg/apis/skupper/v1alpha1/types.go

```diff
@@ -42,7 +42,7 @@ type SkupperClusterPolicySpec struct {
 type Site struct {
 	v1.TypeMeta   `json:",inline"`
 	v1.ObjectMeta `json:"metadata,omitempty"`
-	Spec          SiteSpec `json:"spec,omitempty"`
+	Spec          SiteSpec   `json:"spec,omitempty"`
```

Comment:
These changes seem unrelated to the rest of the PR?

Reply:
These are probably a result of `go fmt` that I ran locally.